docs/operations/installation.md

```diff
@@ -41,6 +41,12 @@ virtualization workloads:
       QEMU: Checking if device /dev/net/tun exists                               : PASS
     ...
 
+### SELinux support
+
+SELinux-enabled nodes need [Container-selinux](https://github.com/containers/container-selinux) installed. The minimum version is documented inside the kubevirt/kubevirt repository, in docs/getting-started.md, under "SELinux support".
+
+For (older) release branches that don't specify a container-selinux version, version 2.170.0 or newer is required.
```

Comment:
`required` -> `recommended`? I mean at least for me it is hard to say what we require in X(pick a random one :) ) without a deeper look.